packages/agents-realtime/package.json

```diff
@@ -11,39 +11,33 @@
   "exports": {
     ".": {
       "browser": {
-        "require": "./dist/index.js",
-        "types": "./dist/index.d.ts",
-        "default": "./dist/index.mjs"
-      },
-      "require": {
         "types": "./dist/index.d.ts",
-        "default": "./dist/index.js"
+        "require": "./dist/index.js",
+        "import": "./dist/index.mjs"
       },
       "types": "./dist/index.d.ts",
-      "default": "./dist/index.mjs"
+      "require": "./dist/index.js",
+      "import": "./dist/index.mjs"
     },
     "./_shims": {
       "workerd": {
-        "require": "./dist/shims/shims-workerd.js",
         "types": "./dist/shims/shims-workerd.d.ts",
-        "default": "./dist/shims/shims-workerd.mjs"
+        "require": "./dist/shims/shims-workerd.js",
+        "import": "./dist/shims/shims-workerd.mjs"
       },
       "browser": {
-        "require": "./dist/shims/shims-browser.js",
         "types": "./dist/shims/shims-browser.d.ts",
-        "default": "./dist/shims/shims-browser.mjs"
+        "require": "./dist/shims/shims-browser.js",
+        "import": "./dist/shims/shims-browser.mjs"
       },
       "node": {
-        "require": "./dist/shims/shims-node.js",
         "types": "./dist/shims/shims-node.d.ts",
-        "default": "./dist/shims/shims-node.mjs"
-      },
-      "require": {
-        "types": "./dist/shims/shims-node.d.ts",
-        "default": "./dist/shims/shims-node.js"
+        "require": "./dist/shims/shims-node.js",
+        "import": "./dist/shims/shims-node.mjs"
       },
-      "types": "./dist/shims/shims-node.d.ts",
-      "default": "./dist/shims/shims-node.mjs"
+      "types": "./dist/shims/shims.d.ts",
```

Comment:
shims/shims is just a reference to shims/shims-node, but there is no downside of this change